src/eval/mod.rs

```diff
@@ -275,8 +280,8 @@ where
 /// # Arguments
 ///
 /// - `clos`: the closure to evaluate
-/// - `global_env`: the global environment containing the builtin functions of the language. Accessible from anywhere in the
-/// program.
+/// - `initial_env`: the initial environment containing the builtin functions of the language.
```

Comment:
The global envrironment doesn't only contain functions. I can't think of a better word here. Small nitpick.